storage/modules/block_storage.go

```diff
@@ -54,6 +54,9 @@ const (
 	// blockSyncIdentifier is the identifier used to acquire
 	// a database lock.
 	blockSyncIdentifier = "blockSyncIdentifier"
+
+	// backwardRelation is a relation from a child to a root transaction
```

Comment:
to add even further clarity, it would be helpful if we mentioned that the root appears in the child's related transactions with a Backward direction. It took me a minute to grasp that at first glance 😅